packs/data/feats.db/grasping-reach.json

```diff
@@ -19,10 +19,42 @@
         "level": {
             "value": 1
         },
+        "location": null,
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Grasping Reach Grip Adjusted",
+                "option": "grasping-reach",
+                "toggleable": true
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "downgrade": true
+                },
+                "predicate": {
+                    "all": [
+                        "melee",
+                        "weapon:hands-held:2",
+                        "grasping-reach"
+                    ],
+                    "any": [
+                        "weapon:damage:die:faces:6",
+                        "weapon:damage:die:faces:8",
+                        "weapon:damage:die:faces:10",
+                        "weapon:damage:die:faces:12"
+                    ],
```

Comment:
This isn't necessary

Reply:
Would you suggest using `"weapon:damage:die:faces:4"` in the `"not"` predicate instead? The feat requires the weapon have a damage die of d6 or larger.

Reply:
Sure, but note also that the weapon must require two hands to be wielded, not merely that it must be held in two hands.

Reply:
Hmm... so exclude weapons with the Two-Hand Dx trait?

Reply:
Updated